Makefile

```diff
@@ -1,5 +1,11 @@
 .PHONY: all lint test init pylint pep8 test-unit test-unit-via-clusterrunner test-functional freeze
 
+NAME=$(shell grep '^Name' .flute | grep -Eo '[[:alnum:]_-]+$$')
+RPM_VERSION_PREFIX=0.5
+COMMIT_COUNT=$(shell git rev-list --count HEAD)
```

Comment:
Fair point. We could work around that however by doing something like converting autoversioning.py to a requirement-less script. So running ./app/util/autoversioning.py would just spit out the version. I think that's better than having two different ways to calculate the version.